docs/en/pid-eaa-issuance.rst

```diff
@@ -536,16 +552,16 @@ The JOSE header of the Wallet Attestation proof of possession MUST contain:
       - **Description**
       - **Reference**
     * - **alg**
-      - A digital signature algorithm identifier such as per IANA "JSON Web Signature and Encryption Algorithms" registry. It MUST be one of the supported algorithms listed in the Section `Cryptographic Algorithms <algorithms.html>`_ and MUST NOT be set to ``none`` or any symmetric algorithm (MAC) identifier.
+      - OPTIONAL. A digital signature algorithm identifier such as per IANA "JSON Web Signature and Encryption Algorithms" registry. It MUST be one of the supported algorithms listed in the Section `Cryptographic Algorithms <algorithms.html>`_ and MUST NOT be set to ``none`` or any symmetric algorithm (MAC) identifier.
       - :rfc:`7516#section-4.1.1`.
     * - **kid**
-      -  Unique identifier of the ``jwk`` inside the ``cnf`` claim of Wallet Attestation as base64url-encoded JWK Thumbprint value.
+      -  OPTIONAL. Unique identifier of the ``jwk`` inside the ``cnf`` claim of Wallet Attestation as base64url-encoded JWK Thumbprint value.
       - :rfc:`7638#section_3`.
     * - **typ**
-      -  It MUST be set to ``jwt-client-attestation-pop``
+      -  OPTIONAL. It MUST be set to ``jwt-client-attestation-pop``
       -  Currently under discussion in [`oauth-attestation-draft <https://vcstuff.github.io/draft-ietf-oauth-attestation-based-client-auth/draft-ietf-oauth-attestation-based-client-auth.html>`_].
 
-The body of the Wallet Attestation proof of possession JWT MUST contain:
+The body of the Wallet Attestation proof of possession JWT contains:
```

Comment:
@cmarco0 please don't change the requirements, the scope of this PR is to only change the HTTP Request POST parameters related to the attestation based client auth to the HTTP Request header parameters as defined in the latest draft.